src/Serilog.Sinks.ApplicationInsights/ExtensionMethods/LogEventExtensions.cs

```diff
@@ -110,11 +110,13 @@ public static class LogEventExtensions
         {
             if (logEvent == null) throw new ArgumentNullException("logEvent");
             if (logEvent.Exception == null) throw new ArgumentException("Must have an Exception", "logEvent");
-            
+
             var exceptionTelemetry = new ExceptionTelemetry(logEvent.Exception)
             {
                 SeverityLevel = logEvent.Level.ToSeverityLevel(),
+#if !NETSTANDARD1_6
```

Comment:
Thanks - yes, as more targets are added, or perhaps when one target is widened, e.g. if AI started supporting .NET Standard 1.5 and we changed the project.json file to accommodate it, these can be a pain to keep updated.